sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -935,7 +929,6 @@ object SparkSession {
         }
 
         session = new SparkSession(sparkContext, None, None, extensions)
-        options.foreach { case (k, v) => session.sessionState.conf.setConfString(k, v) }
```

Comment:
When building `SessionState`, we will merge spark conf to sql conf, which means we don't need to set `options` to `session.sessionState.conf` here, if the spark conf already contains `options`.